WalletWasabi.Fluent/ViewModels/NavBarViewModel.cs

```diff
@@ -17,40 +15,35 @@ namespace WalletWasabi.Fluent.ViewModels
 	/// </summary>
 	public class NavBarViewModel : ViewModelBase
 	{
-		private ObservableCollection<WalletViewModelBase> _items;
 		private ObservableCollection<NavBarItemViewModel> _topItems;
 		private ObservableCollection<NavBarItemViewModel> _bottomItems;
 		private NavBarItemViewModel _selectedItem;
-		private Dictionary<Wallet, WalletViewModelBase> _walletDictionary;
-		private bool _anyWalletStarted;
+		private readonly WalletManagerViewModel _walletManager;
 		private bool _isBackButtonVisible;
-		private IScreen _screen;
 		private bool _isNavigating;
 		private bool _isOpen;
-
 		private Action _toggleAction;
 		private Action _collapseOnClickAction;
 
-		public NavBarViewModel(IScreen screen, RoutingState router, WalletManager walletManager, UiConfig uiConfig)
+
+		public NavBarViewModel(IScreen screen, RoutingState router, WalletManagerViewModel walletManager)
 		{
-			_screen = screen;
 			Router = router;
+			_walletManager = walletManager;
 			_topItems = new ObservableCollection<NavBarItemViewModel>();
-			_items = new ObservableCollection<WalletViewModelBase>();
 			_bottomItems = new ObservableCollection<NavBarItemViewModel>();
 
-			_walletDictionary = new Dictionary<Wallet, WalletViewModelBase>();
-
-			SelectedItem = new HomePageViewModel(screen);
+			var addWalletPage = new AddWalletPageViewModel(screen);
```

Comment:
I have found that it may be useful to use different instance of `AddWalletPageViewModel` for Home and NavBar in case we what do some navigation customization. Right now when you click `Add Wallet` from NavBar there is no back button active.

Reply:
using same instance allows the item to be selected in the navbar too...

lets talk about back buttons offline...

Reply:
> using same instance allows the item to be selected in the navbar too...

ok makes sense